RecoParticleFlow/PFRecHitProducer/interface/PFRecHitParamsRecord.h

```diff
@@ -0,0 +1,14 @@
+#ifndef RecoParticleFlow_PFRecHitProducer_ParamsRecord_h
+#define RecoParticleFlow_PFRecHitProducer_ParamsRecord_h
+
+#include "FWCore/Framework/interface/EventSetupRecordImplementation.h"
+#include "FWCore/Framework/interface/DependentRecordImplementation.h"
+#include "CondFormats/DataRecord/interface/EcalPFRecHitThresholdsRcd.h"
+
+class PFRecHitHCALParamsRecord : public edm::eventsetup::EventSetupRecordImplementation<PFRecHitHCALParamsRecord> {};
+
+class PFRecHitECALParamsRecord
+    : public edm::eventsetup::DependentRecordImplementation<PFRecHitECALParamsRecord,
+                                                            edm::mpl::Vector<EcalPFRecHitThresholdsRcd>> {};
```

Comment:
Why not simply reuse the `EcalPFRecHitThresholdsRcd` instead of defining a new `PFRecHitECALParamsRecord` with the same IOVs ?
Do you foresee adding other dependencies in the future ?

Reply:
This record contains all the parameters that are required to apply cuts to ECAL hits to select which should become PFRecHits. Right now, only a single type of cut is implemented. I think it is possible that other types of cuts will be added in the future.